packages/node_modules/@webex/webex-core/src/lib/services/services.js

```diff
@@ -8,6 +8,7 @@ import ServiceCatalog from './service-catalog';
 import ServiceRegistry from './service-registry';
 import ServiceState from './service-state';
 
+
```

Comment:
This extra new line should be removed.

Reply:
this was an eslint thing... will remove, after SDK owner review.

Reply:
eslint doesn't have a problem with it, so it is good to go!